schema.sql

```diff
@@ -9,3 +9,38 @@ CREATE TABLE animals (
     weight_kg DECIMAL,
     species TEXT
 );
+
+/* owners */
+CREATE SEQUENCE owner_id_seq;
+
+create table owners (
+	id int primary key default nextval('owner_id_seq'),
+	full_name VARCHAR(255),
+	age int
+);
+
```

Comment:
Great job @Abdessittir 🤝

- [ ] As stated in the project requirements, kindly add a new table with the name species with the following columns.
![species](https://user-images.githubusercontent.com/63917725/213413138-a0bf729e-bdc3-4f7b-9786-7db9da3676ba.PNG)